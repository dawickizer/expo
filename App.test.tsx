import React from 'react';
import renderer from 'react-test-renderer';

import App from './App';

describe('<App />', () => {

    // example unit test
    it('has 1 child', () => {
        const tree = renderer.create(<App />).toJSON() as any;
        expect(tree.children.length).toBe(2);
    });

    // example snapshot test
    it('renders correctly', () => {
        const tree = renderer.create(<App />).toJSON();
        expect(tree).toMatchSnapshot();
    });
});


