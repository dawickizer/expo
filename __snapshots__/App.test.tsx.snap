// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> renders correctly 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "backgroundColor": "#25292e",
      "flex": 1,
    }
  }
>
  <View
    style={
      {
        "flex": 1,
        "paddingTop": 58,
      }
    }
  >
    <Image
      source={1}
      style={
        {
          "borderRadius": 18,
          "height": 440,
          "width": 320,
        }
      }
    />
  </View>
  <View
    style={
      {
        "alignItems": "center",
        "flex": 0.3333333333333333,
      }
    }
  >
    <View
      style={
        [
          {
            "alignItems": "center",
            "height": 68,
            "justifyContent": "center",
            "marginHorizontal": 20,
            "padding": 3,
            "width": 320,
          },
          {
            "borderColor": "#ffd33d",
            "borderRadius": 18,
            "borderWidth": 4,
          },
        ]
      }
    >
      <View
        accessibilityState={
          {
            "busy": undefined,
            "checked": undefined,
            "disabled": undefined,
            "expanded": undefined,
            "selected": undefined,
          }
        }
        accessibilityValue={
          {
            "max": undefined,
            "min": undefined,
            "now": undefined,
            "text": undefined,
          }
        }
        accessible={true}
        collapsable={false}
        focusable={true}
        onBlur={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          [
            {
              "alignItems": "center",
              "borderRadius": 10,
              "flexDirection": "row",
              "height": "100%",
              "justifyContent": "center",
              "width": "100%",
            },
            {
              "backgroundColor": "#fff",
            },
          ]
        }
      >
        <Text
          allowFontScaling={false}
          selectable={false}
          style={
            [
              {
                "color": "#25292e",
                "fontSize": 18,
              },
              {
                "paddingRight": 8,
              },
              {
                "fontFamily": "FontAwesome",
                "fontStyle": "normal",
                "fontWeight": "normal",
              },
              {},
            ]
          }
        >
          ï€¾
        </Text>
        <Text
          style={
            [
              {
                "color": "#fff",
                "fontSize": 16,
              },
              {
                "color": "#25292e",
              },
            ]
          }
        >
          Choose a photo
        </Text>
      </View>
    </View>
    <View
      style={
        {
          "alignItems": "center",
          "height": 68,
          "justifyContent": "center",
          "marginHorizontal": 20,
          "padding": 3,
          "width": 320,
        }
      }
    >
      <View
        accessibilityState={
          {
            "busy": undefined,
            "checked": undefined,
            "disabled": undefined,
            "expanded": undefined,
            "selected": undefined,
          }
        }
        accessibilityValue={
          {
            "max": undefined,
            "min": undefined,
            "now": undefined,
            "text": undefined,
          }
        }
        accessible={true}
        collapsable={false}
        focusable={true}
        onBlur={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          {
            "alignItems": "center",
            "borderRadius": 10,
            "flexDirection": "row",
            "height": "100%",
            "justifyContent": "center",
            "width": "100%",
          }
        }
      >
        <Text
          style={
            {
              "color": "#fff",
              "fontSize": 16,
            }
          }
        >
          Use this photo
        </Text>
      </View>
    </View>
  </View>
</View>
`;
